quill-cassandra-ce/src/main/scala-2.12/io/getquill/util/GuavaCeUtils.scala

```diff
@@ -0,0 +1,28 @@
+package io.getquill.util
+
+import com.google.common.util.concurrent.{ FutureCallback, Futures, ListenableFuture }
+
+import cats.effect.Async
+import scala.concurrent.ExecutionContext.global
+
+import java.util.concurrent.Executor
+import scala.language.higherKinds
+
+object GuavaCeUtils {
+  implicit class RichListenableFuture[T](lf: ListenableFuture[T]) {
```

Comment:
I would recommend doing this differently. `ListenableFuture` is an eagerly-evaluating construct, similar to `CompletableFuture` and `Future` itself. The conversion here should really be `F[ListenableFuture[T]] => F[T]`.

Reply:
good idea. Please check the new implementaion.

Reply:
That makes sense. Either way works for me.